examples/internal/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "region" {
+  description = "The name of an AWS region"
+  type        = string
+}
+
+variable "env" {
+  description = "Resource environment tag (i.e. dev, stage, prod)"
+  type        = string
+  default     = "test"
+}
+
+variable "service" {
+  description = "Resource service tag"
```

Comment:
Whilst updating other comments, may as well add `(Optional)` here & to others where its required